recipe/meta.yaml

```
@@ -42,7 +42,7 @@ requirements:
     - {{ cdt('libxdamage') }}  # [linux]
     - {{ cdt('libxxf86vm') }}  # [linux]
   host:
-    - boost =1.68.0
+    - boost 
```

Do you need `Boost.Python`? If not we should use:
```suggestion
    - boost-cpp
```

If so, you should:
- change the `- {{ pin_compatible('boost', max_pin='x.x') }}` line to match
- rerender the feedstock